code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -575,25 +575,25 @@
 	id = "borg_upgrade_rename"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/rename
-	materials = list(MAT_METAL=35000)
-	construction_time = 120
+	materials = list(MAT_METAL = 5000)
+	construction_time = 40
 	category = list("Cyborg Upgrade Modules")
 
 /datum/design/borg_upgrade_restart
 	name = "Cyborg Upgrade (Emergency Reboot Board)"
 	id = "borg_upgrade_restart"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/restart
-	materials = list(MAT_METAL=60000 , MAT_GLASS=5000)
-	construction_time = 120
+	materials = list(MAT_METAL = 20000 , MAT_GLASS = 5000)
+	construction_time = 80
 	category = list("Cyborg Upgrade Modules")
 
 /datum/design/borg_upgrade_vtec
 	name = "Cyborg Upgrade (VTEC Module)"
 	id = "borg_upgrade_vtec"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/vtec
-	materials = list(MAT_METAL=80000 , MAT_GLASS=6000 , MAT_URANIUM= 5000)
+	materials = list(MAT_METAL = 35000 , MAT_GLASS = 12000 , MAT_URANIUM = 10000)
```

Comment:
This seems more suitable now but glass is cheaper than metal. The only question is will miners gather it up more.